### DISCUSSION TITLE
MapExchange not available

### DISCUSSION THREAD

**Discussion by Lasse-Fisker** [urql-graphql/urql#2859] [Q&A] [answered]
Hi! First of all, thanks for the great work on this package 🙏 

I just saw the documentation for `mapExchange` on the docs https://formidable.com/open-source/urql/docs/api/core/#mapexchange, and it seems like the perfect fit for my needs (trying to implement a global loading indicator). 

It seems however, that the exchange is not imported in the `@urql/core` package - perhaps because it is not published on npm yet? If that its the case, does anybody know what the timeline is, and why it's published on the docs if not actually in the package, effectively invalidating the docs in the process? 🙂 

Thanks in advance!

**Comment by JoviDeCroock** [comment, marked as answer]
It's part of the 3.1.0 release which still needs to go out, the docs got released a tad early https://github.com/urql-graphql/urql/pull/2808 - we're looking at one more bugfix before the release

**Reply by Lasse-Fisker** [reply]
Super, that makes sense 👍 I'll wait patiently for the 3.1.0 release then 😄
